app/controllers/course_excluded_exercises_controller.rb

```diff
@@ -0,0 +1,55 @@
+class CourseExcludedExercisesController < ExcludedExercisesController
```

Comment:
I've pondered this for a while now.  I don't think inheritance is really appropriate here, since the base class is not providing any sort of policy for it subclasses (it's just acting as a repository of coincidentally identical code), but I don't think the alternatives (e.g., using a shared library, etc.) are especially more appealing given the situation.

I think we should rename `ExcludedExercisesController` to `ExcludedExercisesControllerBase` to show that it's not a _real_ controller (we should do the same with `JsonApiController`, but in a separate PR).  We might also want to rename `#_response` and `#_definitions` to something like `#_exclusion_response` and `#_exclusion_definitions` to be more clear and to avoid potential naming conflicts.